fhservices.cfg.j2

```diff
@@ -57,26 +57,26 @@ define host {
        host_name {{ rhmap_router_dns }}
 }
 
-{% for fhservice in fh_services_ping %}
+{% for fhservice in fh_services if 'ping' in fhservice.get("checks") %}
```

Comment:
`dict.get` with a single arguments means the default is `None`. It's a `TypeError` to say `if 'ping' in None`.

Are we implicitly treating some dictionary keys as "required"?